src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -502,7 +501,7 @@ def upload_zip_to_storage(cmd, resource_group_name, name, src, slot=None):
 
     container_name = "function-releases"
     blob_name = "{}-{}.zip".format(datetime.datetime.today().strftime('%Y%m%d%H%M%S'), str(uuid.uuid4()))
-
+    BlockBlobService = get_sdk(cmd.cli_ctx, ResourceType.DATA_STORAGE, 'blob#BlockBlobService')
```

Comment:
Can we import it statically like `from azure.storage.blob import BlockBlobService`?
What is the advantage of this approach?

Reply:
Because the latest azure.storage.blob is for track2 without supporting for BlockBlobService, AppendBlobService ..., which are in track1, using azure.multiapi.storage can prevent the breaking change in azure.stroage.blob.